src/DataFetcher.py

```diff
@@ -229,9 +230,18 @@ def parse_yahoo_finance_analysis(self, response, *args, **kwargs):
   def fetch_yahoo_finance_quote(self):
     self.yahoo_finance_quote = YahooFinanceQuote(self.ticker_symbol)
     session = self._create_session()
+    # set_cookie = requests.get("https://fc.yahoo.com", allow_redirects=True)
+    # cookie = {"A3": "=d=AQABBDIjb2QCEP6Yfv8deeHPmCkMOJPr4LYFEgEBAQF0cGR5ZO-V7L8A_eMAAA&S=AQAAAtfWKlJiCqSO9jMVNukiJs8"}
+    # print(set_cookie.cookies)
+    # crumb = requests.get("https://query2.finance.yahoo.com/v1/test/getcrumb", cookies=set_cookie.cookies)
+    # print(crumb.headers)
+    # print(crumb.__dict__)
```

Comment:
Can we remove all of these commented out lines?

Actually it looks like this file wasn't even really touched. Can we just revert it?

Reply:
Reversed